src/clover/__main__.py

```diff
@@ -1053,23 +1043,22 @@ def main(  # pylint: disable=too-many-locals, too-many-statements
             )
 
             if parsed_args.analyse:
-                if not parsed_args.skip_plots:
```

Comment:
## :rewind: Revert this
It looks like this change has somehow been reverted on your branch somehow? We want to keep this `parsed_args.skip_plots` code in 😄

Reply:
So we want to keep from line 1056 to line 1072 in? and the rest right?

Reply:
We basically want it how it is in red on the Github, not the "green" changes :smile: